pkg/apis/resource/types.go

```diff
@@ -1210,6 +1342,14 @@ type DeviceAllocationResult struct {
 // entries in allocation.devices.results.
 const AllocationResultsMaxSize = 32
 
+// Byte length of random ID for ShareUID; String length will be twice characters.
+const ShareIDNBytes = 3
+const ShareIDLength = ShareIDNBytes * 2
+
+// Max length for shared device name, accounting for suffix format "-[ShareUID]",
+// where ShareUID is 2 * ShareUIDNBytes characters.
+const SharedDeviceNameMaxLength = DeviceMaxDomainLength - ShareIDNBytes - 1
```

Comment:
This calculation is incorrect. ShareID uses 6 characters (ShareIDLength), not 3 bytes.
```suggestion
const SharedDeviceNameMaxLength = DeviceMaxDomainLength - ShareIDLength - 1
```

Reply:
@utam0k Thank you! updated.